### DISCUSSION TITLE
Remembering position per file.

### DISCUSSION THREAD

**Discussion by ThomasLeigh** [PaulWoitaschek/Voice#1502] [Ideas]
I often use audiobook apps for listening podcasts also. It would be fantastic if the app could be able to remember listening position per file. For example, if I pause listening to the first file on 50% and switch to the other file within the same folder ("audiobook") and listen 30% of that file, and then I switch back to the previous file - it should be paused on 50%, and then if I switch to the second file it should be paused on 30%. 

If this ability would be all right for You it would make a great difference.

**Comment by ueen** [comment]
You could just let Voice display every file as it's own audiobook, what you should be doing is getting AtennaPod that is specifically designed for Podcasts :)
